configurations/arm/ubuntu2204.yaml

```diff
@@ -0,0 +1,8 @@
+ami_db_scylla_user: 'ubuntu'
+ami_id_db_scylla: 'ami-0127b718f588563a0'
```

Comment:
Can you add the name of the ami in the comments?

Reply:
I can't find this ami for some reason... I'm thinking if we even need this. We're providing the `scylla_repo` param in artifacts tests and these 2 should not be used together either way, IIUC. @roydahan Are these values in `/configurations` necessary?

Reply:
since we are running in AWS, we need this as the base image, regardless if we using `scylla_repo` to install or not.
this image is `ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-20200701` (it's public image)
but it's not 22.04 as this PR titled.

you probably want to use this:
```yaml
ami_id_db_scylla: ami-022c8ce295ce9ac4c  # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-arm64-server-20220506 
```

Reply:
Ah, that's correct. I added the base image and comment.